### PR TITLE
Minor: update the agent docker command after the onboarding support

### PR DESCRIPTION
**Pull request by ctan888**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ The agent application can be configured via the use of several environment varia
 | Name                                            | Description                                                                      | Required | Default                       |
 | ----------------------------------------------- | -------------------------------------------------------------------------------- | -------- | ----------------------------- |
 | SUPERBLOCKS_AGENT_ID                            | UUID used by Superblocks Cloud to identify the agent                             | Yes      | n/a                           |
+| SUPERBLOCKS_AGENT_KEY                           | Secret key used by Superblocks Cloud to authorize the agent                      | Yes      | n/a                           |
+| SUPERBLOCKS_AGENT_HOST_URL                      | URL used by end-users to access the agent                                        | Yes      | n/a                           |
```
Comment (taha-au) [inline README.md:13]: SUPERBLOCKS_AGENT_HOST_URL defaults to `http://localhost:8020/agent`

**Review by taha-au — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (taha-au) [reply README.md:13]: Change the description to the following
```
URL used by end-users to access the agent; should be overridden for non-localhost deployments
```

Change required to No

**Review by taha-au — CHANGES_REQUESTED** [review]

**Comment by taha-au** [comment]
@ctan888 Also update the docker run and docker compose commands in the appropriate sections

**Comment by ctan888** [comment]
> @ctan888 Also update the docker run and docker compose commands in the Deployment section

I think I already have them updated. Let me know something specific is missing here.

**Review by taha-au — APPROVED** [review]

**Pull request merged**